wagtail/users/views/bulk_actions/toggle_activity.py

```diff
@@ -45,7 +45,10 @@ def prepare_action(self, objects):
     @classmethod
     def execute_action(cls, objects, mark_as_active=False, model=None, **kwargs):
         if model is None:
-            return
+            if len(cls.models) == 1:
```

Comment:
This feels like this could end up in a lot of actions - maybe something to factor out to reduce the boilerplate a bit?

Reply:
Additionally, does this need to go in a few more actions here? Feels like this should be applicable to more actions